libiocage/lib/Jail.py

```diff
@@ -18,6 +18,49 @@
 
 
 class Jail:
+    """
+    Iocage unit orchestrates a jail's configuration and manages state
```

Comment:
Perhaps say `iocage Jail unit`? Remember `iocage` is always lowercase, it's our name. That goes for all over.